### ISSUE TITLE
ERROR: Failed building wheel for hdbscan; Microsoft C++ Build tools missing

### ISSUE THREAD

**Issue by JensBlack** [YttriLab/A-SOID#59]
Some windows users might experience the following issue when installing A-SOiD (see below). This is connected to building tools missing and can be solved by installing the Microsoft C++ Build tools.

Issue (extract):
````
ERROR: Failed building wheel for hdbscan
ERROR: Could not build wheels for hdbscan, which is required to install pyproject.toml-based projects
failed
````

Solution
````
Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/      
````

Long Issue:
````                                                                                                                                                                                                                                                                                                         error: subprocess-exited-with-error
× Building wheel for hdbscan (pyproject.toml) did not run successfully.
│ exit code: 1
...
error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/                                                          
[end of output]
note: This error originates from a subprocess, and is likely not a problem with pip.
ERROR: Failed building wheel for hdbscan
ERROR: Could not build wheels for hdbscan, which is required to install pyproject.toml-based projects failed
CondaEnvException: Pip failed
````

**Comment by fmassaNU** [comment]
I installed the build tools as suggested, but I'm not sure what to do next. I tried running the env create command again and it says "CondaValueError: prefix already exists". I successfully activated the environment, but I was unable to run the app. The error message upon trying to run the app was "the term 'asoid' is not recognized as the name of a cmdlet, function, script file, or operable function."
